apps/video/test/marionette/control_test.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var Video = require('./lib/video'),
+    assert = require('assert');
+    
+marionette('video info view', function() {
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true
+      }
+    },
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var app, selectors, initialTime;
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+    
+    // Add file into the videos directory
+    client.fileManager.add({
+      type: 'videos',
+      filePath: 'test_media/Movies/elephants-dream.webm'
+    });
+
+    selectors = Video.Selector;
+    app = new Video(client);
+  });
+
+  test('open video and check video info', function() {
+    // tap thumbnail to open video
+    app.launch();
+    app.waitForThumbnails(1);
+    app.tapThumbnail(0);
+    app.waitForVideoToStartPlaying();
+    
+    // pause, then tap forward and check the counter 
+    // that it was increased by 10 seconds
+    app.pausePlayback();
+    initialTime = app.getElapsedTimeSeconds();
+    app.tapForward();
+    assert.ok(app.getElapsedTimeSeconds() == (initialTime + 10));
```

Comment:
Nit: To reduce the number of times the test is calling 'getElapsedTimeSeconds', perhaphs this code could be something like:
app.pausePlayback();
initialTime = app.getElapsedTimeSeconds();
app.tapForward();
var nextTime = app.getElapsedTimeSeconds();
assert.ok((nextTime - 10) == initialTime);
initalTime = nextTime;
app.tapRewing();
nextTime = app.getElapsedTimeSeconds();
assert.ok((nextTime + 10) == initialTime);
initialTime = nextTime ...

If you think this is overly complicated, feel free to leave the code as it is.